# CONFIG FOR THE ROW DETECTOR
one_line: false            # If true fits one line to the eliptical region (suitable for uv scenario) else fits a line in each half of eliptical region (suitable for rasberry scenario).               
rx: 7.0                    # Radius (meters) in x of elipse
ry: 1.5                    # Radius (meters) in y of elipse 
eps: 0.15                  # Parameter of DBSCAN clustering algorithm. The maximum distance (meters) between two samples for one to be considered as in the neighborhood of the other.        
min_samples_dbscan: 2      # Parameter of DBSCAN clustering algorithm. The number of samples in a neighborhood for a point to be considered as a core point.            
min_samples_ransac: 3      # Parameter of ransac linear regression. Minimum number of samples chosen randomly from original data.              
residual_threshold: 0.2    # Parameter of ransac linear regression. Maximum residual for a data sample to be classified as an inlier.        
num_attempted_fits: 3      # Number of attempts allowed for ransac to fit a line
line_granularity: 1000     # Granularity of fitted line
min_num_poles: 0           # turn off svm filtering if less than min_num_poles found in a scan. Set to zero to always filter with svm.
